docs/csharp/whats-new/csharp-7-1.md

```diff
@@ -0,0 +1,187 @@
+---
+title: What's new in C# 7.1
+description: An overview of new features in C# 7.1.
+keywords: C# language design, 7.1, Visual Studio 2017, 
+author: billwagner
+ms.author: wiwagn
+ms.date: 08/16/2017
+ms.topic: article
+ms.prod: .net
+ms.devlang: devlang-csharp
+---
+
+# What's new in C# 7.1
+
+C# 7.1 is the first point release to the C# Language. It marks an increased
+release cadence for the language. You can use the new features sooner, ideally
+when each new feature is ready. C# 7.1 adds the ability to configure
+the compiler to match a specified version of the language. That enables you to
+separate the decision to upgrade tools from the decision to upgrade language
+versions.
+
+C# 7.1 adds the [language version selection](#language-version-selection)
+configuration element, and four new language features.
+
+The new language features in this release are:
+
+* [`async` `Main` method](#async-main)
+  - The entry point for an application can have the `async` modifier.
+* [`default` literal expressions](#default-literal-expressions)
+  - You can use default literal expressions in default value expressions when the target type can be inferred.
+* [Inferred tuple element names](#inferred-tuple-element-names)
+  - The names of tuple elements can be inferred from tuple initialization in many cases.
+* [Reference assembly generation](#reference-assembly-generation)
```

Comment:
Technically that's not a language feature (it's just a compiler feature), but I don't think it's worthwhile to change.